code/datums/diseases/xeno_transformation.dm

```diff
@@ -36,18 +37,36 @@
 			if (prob(10))
 				to_chat(affected_mob, pick(SPAN_DANGER("Your skin feels very tight."), SPAN_DANGER("Your blood boils!")))
 				affected_mob.take_limb_damage(3)
+				playsound(affected_mob.loc, 'sound/voice/alien_growl1.ogg', 15, FALSE)
 			if (prob(5))
-				affected_mob.whisper(pick("Soon we will be one...", "Must... devour...", "Capture...", "We are perfect.", "Hsssshhhhh!"))
+				if(!ismonkey(affected_mob))
+					affected_mob.whisper(pick("Soon we will be one...", "Must... devour...", "Capture...", "We are perfect.", "Hsssshhhhh!"))
 			if (prob(8))
 				to_chat(affected_mob, SPAN_DANGER("You can feel... something...inside you."))
 		if(5)
 			to_chat(affected_mob, SPAN_DANGER("Your skin feels impossibly calloused..."))
 			affected_mob.apply_damage(10, TOX)
 			affected_mob.updatehealth()
 			if(prob(40))
-				var/turf/T = find_loc(affected_mob)
-				gibs(T)
-				var/mob/living/carbon/human/H = affected_mob
-				H.Alienize(XENO_T1_CASTES)
+				var/turf/turf = get_turf(affected_mob)
+				var/area/area = get_area(turf)
+				gibs(turf)
+				if(ismonkey(affected_mob))
+					new /mob/living/simple_animal/hostile/alien/ravager(turf)
```

Comment:
i would recomend you not doing this ? pretty sure mob xenos are not supposed to be in normal gameplay

Reply:
they work fine though? their AI works and they target and slash humans

Reply:
turned into feral xeno